swan-lake/development-tutorials/ballerina-persist/supported-data-stores.md

```diff
@@ -221,3 +222,70 @@ Follow the steps below to obtain API tokens for Google Sheets.
     3. For step 1 under the  `Select & authorize APIs` section, select the required scopes for the API you are going to use. 
     4. Proceed with the necessary Google account validation steps. 
     5. Click **Exchange authorization code for tokens** to obtain the refresh and access tokens.
+
+## Redis [Experimental]
+
+The Redis data store is a key-value based inmemory datastore. It is useful for storing unstructured data and retrieve them quickly. It is not the default data store for `bal persist`. Therefore, you need to explicitly specify the data store when initializing `bal persist` in your application as follows.
+
+```shell
+$ bal persist init --datastore redis
+```
+
+### Supported Ballerina types
+
+The Redis data store supports the following Ballerina types. As we have utilized `hash` and `set` as Redis data types to store objects, the following data types will be converted to `string`.
+
+- int
+- float
+- decimal
+- string
+- time:Date
+- time:TimeOfDay
+- time:Utc
+- time:Civil
+
+### Configuration
+
+You need to set values for the following basic configuration parameters in the `Config.toml` file in your project to use the Redis data store.
+
+| Parameter  |             Description              |
+|:----------:|:------------------------------------:|
+| connection | The connection URI of the DB server. |
+
+The following is a sample `Config.toml` file with the Redis data store configuration. This is generated by the `persist generate` command.
+
+```toml
+[<packageName>.<moduleName>.redis]
+connection = "redis://localhost:6379"
+```
+
+Alternatively, you can provide connection parameters instead of `URI` as follows.
+```toml
+[<packageName>.<moduleName>.redis.connection]
+host = "localhost"
+port = 6379
+```
+
+Additionally, you can set values for the advanced configuration parameters in the Config.toml file in your project to use the Redis data store. For more information on these parameters, see the [Redis Connector documentation](https://central.ballerina.io/ballerinax/redis/latest#ConnectionConfig).
+
+### How to set up
```

Comment:
Would it be better use the heading as `Setup guide`?

Reply:
Yes, It's better to have that. I have addressed this in https://github.com/ballerina-platform/ballerina-dev-website/pull/8838/commits/29131f97d9abca7093bac3ef255af9635d96edc4